x-pack/test/api_integration/apis/ml/calculate_model_memory_limit.ts

```diff
@@ -57,7 +57,7 @@ export default ({ getService }: FtrProviderContext) => {
           detectors: [
             {
               function: 'avg',
-              field_name: 'geoip.city_name',
+              field_name: 'taxless_total_price',
```

Comment:
Nit, but seeing as we are editing this test, amend the title to `with 1 metric and 1 influencer same as split field`

Reply:
Done in c9d8a00d30